docs/diagrams/data-flow/data-flow-api-endpoint.puml

```diff
@@ -0,0 +1,153 @@
+'
+'  Copyright (c) 2021 Daimler TSS GmbH
+'
+'  This program and the accompanying materials are made available under the
+'  terms of the Apache License, Version 2.0 which is available at
+'  https://www.apache.org/licenses/LICENSE-2.0
+'
+'  SPDX-License-Identifier: Apache-2.0
+'
+'  Contributors:
+'      Daimler TSS GmbH - Initial Version
+'
+
+@startuml
+
+skinParam NoteBackgroundColor WhiteSmoke
+skinParam NoteFontColor Black
+skinParam ParticipantBackgroundColor WhiteSmoke
+skinParam ActorBackgroundColor WhiteSmoke
+skinParam AgentBackgroundColor White
+skinParam AgentBorderColor SkyBlue
+skinparam shadowing false
+
+!define ConsumerColor f8f2ff
+!define ProviderColor d9edff
+!define WarningColor Business
+!define LeadColor Technology
+
+autonumber
+
+box Consumer #ConsumerColor
+    participant CBE as "Backend Service"
+    participant CCP as "Control Plane" #Business
+    participant CDP as "Data Plane" #Business
+end box
+
+box Provider #ProviderColor
+    participant PDP as "Data Plane" #Business
+    participant PCP as "Control Plane" #Business
+    participant PBE as "Backend Service"
+end box
```

Comment:
If you want to make it clearer that the EDC and BDS are two different services, as we properly would suggest, you could split the Consumer and Provider boxes into two each.

```suggestion
box "Consumer\n controlled zone" #ConsumerColor
    participant CBE as "Backend Service"
end box

box Consumer EDC #ConsumerColor
    participant CCP as "Control Plane" #Business
    participant CDP as "Data Plane" #Business
end box

box Provider EDC #ProviderColor
    participant PDP as "Data Plane" #Business
    participant PCP as "Control Plane" #Business
end box

box "Provider\n controlled zone" #ProviderColor
    participant PBE as "Backend Service"
end box
```

This would look like this:
![image](https://user-images.githubusercontent.com/97295414/151502222-23630c3f-ab2d-43f5-9156-8dec6d165591.png)